tests/std/tests/P0896R4_ranges_alg_uninitialized_move_n/test.cpp

```diff
@@ -236,7 +236,7 @@ struct memcpy_test {
 
 template <test::ProxyRef IsProxy>
 using test_input  = test::range<test::input, int_wrapper, test::Sized::no, test::CanDifference::no, test::Common::no,
-    test::CanCompare::no, IsProxy>;
+     test::CanCompare::no, IsProxy>;

```

Comment:
why this extra space?

Reply:
clang-format 18 has interesting new ideas about how to indent wrapped code. I don't entirely understand its new heuristics yet.